examples/api-basic/api/v1/operators/converter.go

```diff
@@ -0,0 +1,54 @@
+package operators
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/caicloud/nirvana/definition"
+	"github.com/caicloud/nirvana/examples/api-basic/application"
+)
+
+func operater(f interface{}) definition.Operator {
+	value := reflect.ValueOf(f)
```

Comment:
It is not good for a framework to make user write `reflect` code...especially in their business code lines.

Reply:
It's an example!

Reply:
Example should be easy to read for potential users...